### DISCUSSION TITLE
Help: about using .lazy.lua

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3664] [Q&A]
https://www.lazyvim.org/news#12x

The latest release notes mention the `.lazy.lua` configuration file, but I can't see any processing for it from the source code. And don't know how to use it

One [pr](https://github.com/LazyVim/LazyVim/pull/2115) that was submitted, lazyrc.lua, seems to be missing.

**Comment by dotfrag** [comment]
LazyVim doesn't do the processing of `.lazy.lua`, `lazy.nvim` does. From the [README](https://github.com/folke/lazy.nvim) default plugin opts:

```lua
  local_spec = true, -- load project specific .lazy.lua spec files. They will be added at the end of the spec.
```

**Reply by iamxiaojianzheng** [reply]
Is there a place I can learn how to configure.lazy.lua? What are the configuration items?

**Reply by dpetka2001** [reply]
@iamxiaojianzheng Just create a file `.lazy.lua` at the root of your project just like the files that you create in your `/lua/plugins/` directory. It should just return specs of plugins.
